### PR TITLE
Added CPM value as parameter for Vertoz Adapter

### PR DESCRIPTION
**Pull request by Prebid-Vertoz**
## Type of change

- [x ] Other

## Description of change

Addition of CPM value as a parameter in Vertoz Adapter.

### CODE REVIEW

**Comment by Prebid-Vertoz** [comment]
Hi team,

Just to ensure, is anything more needs to be done from our side?

**Comment by mkendall07** [comment]
@Prebid-Vertoz 
Please rebase off master as the adapters are now under `/modules` instead of `/src/adapters`

**Comment by Prebid-Vertoz** [comment]
@mkendall07 Thanks for your reply.
I have done changes now.

File: modules/vertozBidAdapter.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ function VertozAdapter() {
       let vzEndPoint = BASE_URI;
       let reqParams = bid.params || {};
       let placementId = utils.getValue(reqParams, 'placementId');
+      let cpm = utils.getValue(reqParams, 'cpm');
```
Comment (mkendall07) [inline modules/vertozBidAdapter.js]: what is `cpm` suppose to represent? Is this a floor?

Comment (Prebid-Vertoz) [inline modules/vertozBidAdapter.js]: Yes, it is a floor.

Comment (mkendall07) [inline modules/vertozBidAdapter.js]: can you rename it to cpmFloor or similar than?

Comment (Prebid-Vertoz) [inline modules/vertozBidAdapter.js]: Hi @mkendall07 , renamed cpm to cpmFloor.

**Comment by mkendall07** [comment]
this is merged. Please update the documentation page as well. Thanks

**Comment by Prebid-Vertoz** [comment]
Hi @mkendall07 we have updated documentation page for this change and placed a pull request.
But we could not see our change reflected when we downloaded prebid.js from download page with only vertoz adapter selected.

**Comment by mkendall07** [comment]
It will be released next week when we update the download page.

**Comment by Prebid-Vertoz** [comment]
Hi @mkendall07 , is that download page updated. Actually we still can not see our change reflected when we downloaded prebid.js with vertoz adapter.

**Comment by Prebid-Vertoz** [comment]
Hi @mkendall07 , any update on this?

**Pull request merged**